engine/class_modules/sc_rogue.cpp

```diff
@@ -1203,6 +1243,8 @@ class rogue_t : public player_t
     if ( set_bonuses.t31_outlaw_2pc->ok() )
       proc_chance += set_bonuses.t31_outlaw_2pc->effectN( 1 ).percent();
     proc_chance += buffs.skull_and_crossbones->stack_value();
+    if ( talent.fatebound.destiny_defined->ok() )
+      proc_chance += talent.fatebound.destiny_defined->effectN( 1 ).percent();
```

Comment:
Effect 1 is clearly the poison proc rate, and effect 2 (and 3!) is also a value 5 effect with a dummy label and server side script tag. I should probably operate on the assumption effect 2 is supposed to be the opportunity proc rate addition, rather than reusing effect 1 here (and that effect 3 is the bonus coinflip matching rate).